docs/management/admin/event-filters.asciidoc

```diff
@@ -12,25 +12,44 @@ By default, an event filter is recognized globally across all hosts running {end
 
 Create event filters from the Hosts page or the Event filters page.
 
-. To create an event filter from the Hosts page:
+. Do one of the following:
++
+--
+* To create an event filter from the Hosts page:
 .. Go to *Explore* -> *Hosts*.
 .. Select the *Events* tab to view the Events table.
 +
-.. Find the event to create a filter, click the *More actions* button (*...*), then click *Add Endpoint event filter*.
+.. Find the event to filter, click the More actions menu (*...*), then select *Add Endpoint event filter*.
```

Comment:
```suggestion
.. Find the event to filter, click the *More actions* menu (*...*), then select *Add Endpoint event filter*.
```

Reply:
Also, it looks like we refer to this as a button in the "Manage detection alerts" topic.

Reply:
Good catch! @nastasha-solomon Since you're currently editing that topic for https://github.com/elastic/security-docs/pull/1836, could you revise that instance of "button" into "menu"?